src/test/java/web/tests/FlipkartTest.java

```diff
@@ -25,12 +31,110 @@ public void testSetUp() {
+
+        SearchPage searchPage = new HomePage().popUpCancel().setShoes(product).searchShoes().sortShoes("Price -- Low to High");
+
+        /**
+         * Getting the product name and price of shoe and storing in array
+         */
+        for (int row = 0; row < shoeno.length; row++) {
+            for (int col = 0; col < 2; col++) {
+                productNameAndPrice[row][col] = searchPage.getProductName(shoeno[row]);
+                col = col + 1;
+                productNameAndPrice[row][col] = searchPage.getProductPrice(shoeno[row]);
+            }
+        }
+
+        /**
+         * Just printing product Name And Price array
+         */
+        for (int row = 0; row < shoeno.length; row++) {
+            for (int col = 0; col < 2; col++) {
+                logger.info("****productNameAndPrice = *** = " + productNameAndPrice[row][col]);
+            }
+        }
+
+        /**
+         * Selecting shoe 1 and adding in cart , same procedure for other shoes
+         */
+        CartPage cartPage = null;
+        for (int index = 0; index < shoeno.length; index++) {
+            cartPage = searchPage.OpenProductPage(shoeno[index]).clickShoeSize().addToCartShoe();
+            /**
+             * If you are selecting last shoe than below if statement will not run, it will not switch to parent window
+             */
+            if (!((shoeno.length - 1) == index)) {
+                switchParentWindow();
+            }
+        }
+
+        /**
+         * getting ProductName And Price From AddtoCart
+         */
+        String[][] getProductNameAndPriceFromAddtoCart = new String[shoeno.length][shoeno.length];
+        for (int row = 0; row < shoeno.length; row++) {
+            for (int col = 0; col < 2; col++) {
+                int positionOfProductName = row + 1;
+                int positionOfPrice = positionOfProductName + 1;
+                getProductNameAndPriceFromAddtoCart[row][col] = cartPage.getProductName(positionOfProductName);
+                col = col + 1;
+                getProductNameAndPriceFromAddtoCart[row][col] = cartPage.getProductPrice(positionOfPrice);
+            }
+        }
+
+        /**
+         * Just printing the array and seeing the value
+         */
+        for (int row = 0; row < getProductNameAndPriceFromAddtoCart.length; row++) {
+            for (int col = 0; col < 2; col++) {
+                logger.info("****getProductNameAndPriceFromAddtoCart = *** = " + getProductNameAndPriceFromAddtoCart[row][col]);
+            }
+        }
+
+        /**
+         * Now comparing 2 array product shoes, not price
+         */
+        for (int row = 0; row < productNameAndPrice.length; row++) {
+
+            String singelProductNameAndPriceFromList = productNameAndPrice[row][0];
+            int counter = 0;
+
+            for (int rowToCheckProductInAddToCart = 0; rowToCheckProductInAddToCart < getProductNameAndPriceFromAddtoCart.length; counter = rowToCheckProductInAddToCart++) {
```

Comment:
I think we can remove AddToCart suffix in rowToCheckProductInAddToCart - since this is already a test for AddtoCart. Will also shorten the variable names